spec/features/admin/index_spec.rb

```diff
@@ -62,22 +61,55 @@
     @transaction_6 = Transaction.create!(credit_card_number: "5", result: 0, invoice_id: @invoice_6.id)
     @transaction_7 = Transaction.create!(credit_card_number: "5", result: 0, invoice_id: @invoice_7.id)
     @transaction_8 = Transaction.create!(credit_card_number: "5", result: 0, invoice_id: @invoice_8.id)
-    # binding.pry
-
   end
+  
+  describe 'On admin dashboard we see top 5 customers' do
     it 'displays top 5 customers' do
-     visit "/admin"
+      visit "/admin"
       # save_and_open_page
-     expect(page).to have_content("Eli Fuchsman: 1")
-     expect(page).to have_content("Darby Smith: 1")
-     expect(page).to have_content("William Lampke: 1")
-     expect(page).to have_content("Bryan Keener: 3")
-     expect(page).to have_content("James White: 2")
+      expect(page).to have_content("Eli Fuchsman: 1")
+      expect(page).to have_content("Darby Smith: 1")
+      expect(page).to have_content("William Lampke: 1")
+      expect(page).to have_content("Bryan Keener: 3")
+      expect(page).to have_content("James White: 2")
+
+      expect("Bryan Keener: 3").to appear_before("James White: 2")
+      expect("James White: 2").to appear_before("Eli Fuchsman: 1")
+      expect("Eli Fuchsman: 1").to appear_before("Darby Smith: 1")
+      expect("Darby Smith: 1").to appear_before("William Lampke: 1")
+    end
+  end
+
+  describe 'admin dashboard incomplete invoices' do
+    it 'has a section for incomplete invoices that contains ids of invoices for items that have not been shipped' do
+      visit "/admin"
+      
+      within("#incomplete_invoices") do
+        expect(page).to have_link("#{@invoice_3.id}")
+        expect(page).to have_link("#{@invoice_1.id}")
+        expect(page).to have_link("#{@invoice_4.id}")
+        expect(page).to have_link("#{@invoice_5.id}")
+        expect(page).to_not have_link("#{@invoice_2.id}")
+      end
+    end
+    
+    it 'orders invoices from oldest to newest by creation date' do
+      visit "/admin"
+
+      within("#incomplete_invoices") do
+        expect("#{@invoice_3.id}").to appear_before("#{@invoice_1.id}")
+        expect("#{@invoice_1.id}").to appear_before("#{@invoice_4.id}")
+        expect("#{@invoice_4.id}").to appear_before("#{@invoice_5.id}")
+      end
+    end
+
+    it 'shows the creation date for incomplete invoices in a Day, Month Date, Year' do
+      visit "/admin"
 
-     expect("Bryan Keener: 3").to appear_before("James White: 2")
-     expect("James White: 2").to appear_before("Eli Fuchsman: 1")
-     expect("Eli Fuchsman: 1").to appear_before("Darby Smith: 1")
-     expect("Darby Smith: 1").to appear_before("William Lampke: 1")
+      expect(page).to have_content("Saturday, October 15, 2022")
```

Comment:
At some point we will need to refactor to utilize `within` blocks to for a 3 in Testing and Debugging on the rubric.